react_demo/index.html

```diff
@@ -7,8 +7,8 @@
   </head>
   <body>
     <div id="c5t-root">Loading...</div>
-    <script src="./vendor/react-0.13.3-build/react.js"></script>
-    <script src="./vendor/react-0.13.3-build/JSXTransformer.js"></script>
+    <script src="./vendor/react/react.js"></script>
```

Comment:
Alright, I suggest we close this pull request without merging then. I have
a functional demo locally, and real code using it is coming soon.

On Saturday, June 6, 2015, John Babak <notifications@github.com> wrote:

> In react_demo/index.html
> <https://github.com/KnowSheet/SimpleDashboard/pull/7#discussion_r31874053>
> :
>
> > @@ -7,8 +7,8 @@
> >    </head>
> >    <body>
> >      <div id="c5t-root">Loading...</div>
> > -    <script src="./vendor/react-0.13.3-build/react.js"></script>
> > -    <script src="./vendor/react-0.13.3-build/JSXTransformer.js"></script>
> > +    <script src="./vendor/react/react.js"></script>
>
> Then delete the original (my) files to avoid such confusions. I'm already
> confused by jumping to commit hashes and then not seeing that committed
> changes in this PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KnowSheet/SimpleDashboard/pull/7/files#r31874053>.
>


-- 
And, as always, have a nice day!
Dima

http://dimakorolev.com/ | http://www.linkedin.com/in/dimakorolev |
dmitry.korolev@gmail.com | +1 (312) 593 2783